resources/js/user/components/ConfirmPopup.vue

```diff
@@ -0,0 +1,74 @@
+<template>
+    <div class="modal fade" tabindex="-1" role="dialog" aria-labelledby="mySmallModalLabel" aria-hidden="true"
+        id="mi-modal">
+        <div class="modal-dialog">
+            <div class="modal-content">
+                <div class="modal-header">
+                    <h4 class="modal-title" id="myModalLabel">{{ titleConfirm }}</h4>
+                    <button type="button" class="btn-close btn-dark" data-dismiss="modal" aria-label="Close"
+                        data-bs-dismiss="modal"></button>
+                </div>
+                <div class="modal-footer">
+                    <button type="button" class="btn btn-light" id="modal-btn-yes" @click.prevent="confirmYes()">{{ btnYes
+                    }}</button>
+                    <button type="button" class="btn btn-primary" id="modal-btn-no" @click.prevent="confirmNo()">{{ btnNo
+                    }}</button>
+                </div>
+            </div>
+        </div>
+    </div>
+</template>
+
+<script>
+export default {
+    name: "ConfirmPopup",
+    data() {
+        return {
+            titleConfirm: 'Do you want to delete this user?',
+            btnYes: 'Yes',
+            btnNo: 'No',
+            timeDelay: 2000,
+        }
+    },
+    props: ['user-id', 'url-users'],
+    methods: {
+        confirmNo: function () {
+            $("#mi-modal").modal('hide');
+        },
+        confirmYes: function () {
+            if (typeof this.userId === "undefined" || this.userId === null) {
+                console.error("Error deleting post: Not defined post");
+            } else {
+                this.deleteUser();
+            }
+        },
+        deleteUser: function () {
+            this.axios.delete(`/admin/users/delete_user/${this.userId}`)
+                .then(response => {
+                    $('#messageAjax').html(response.data.message)
+                    const toastClass = response.data.status === 'success' ? 'text-bg-success' : 'text-bg-error';
+                    $("#liveToast").addClass(toastClass);
+                    this.showToast();
+                    if (typeof this.urlUsers === "undefined" || this.urlUsers === null) {
```

Comment:
this.urlUsers này khai báo ở đâu á e

Reply:
Dạ urlUsers em sử dụng với chức năng delete trong Page Detail User 
![image](https://github.com/anhlp1212/blog_laravel/assets/147794362/962aafb9-1be4-4c38-b080-5a6a2afad440)
Em truyền biến urlUsers ở đây để sau khi xóa sẽ chuyển sang Page List User, cái này sẽ rõ hơn ở branch
feature/BogPage-Laravel-ManageUserVuejs-DetailUser ạ (em chưa tách source branch này ạ)